src/SWP/ContentBundle/Tests/Twig/Extension/ContentExtensionTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Content Bundle.
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\ContentBundle\Tests\Twig\Extension;
+
+use Liip\FunctionalTestBundle\Test\WebTestCase;
+use SWP\ContentBundle\Twig\Extension\ContentExtension;
+use SWP\ContentBundle\Document\Article;
+use SWP\TemplatesSystem\Gimme\Meta\Meta;
+
+class ContentExtensionTest extends WebTestCase
+{
+    public function testGetFunctions()
+    {
+        $contentExtension = new ContentExtension(
+            $this->getContainer()->get('doctrine.orm.entity_manager'),
+            $this->getContainer()->get('router')
+        );
+        $functions = $contentExtension->getFunctions();
+
+        $generateUrlFor = false;
+        foreach ($functions as $name => $function) {
+            if ($function instanceof \Twig_SimpleFunction) {
+                if ($function->getName() == 'generateUrlFor*') {
+                    $generateUrlFor = true;
+                }
+            }
+        }
+
+        $this->assertTrue($generateUrlFor, 'Should find "generateUrlFor*" to be a valid function');
+    }
+
+    public function testGetName()
+    {
+        $contentExtension = new ContentExtension(
+            $this->getContainer()->get('doctrine.orm.entity_manager'),
+            $this->getContainer()->get('router')
+        );
+        $this->assertEquals('swp_content', $contentExtension->getName(), 'Should have name "gitHubGist"');
```

Comment:
`'Should have name "gitHubGist"'` ? should be be `"swp_content"` not `"gitHubGist"`